src/components/common/Input/index.js

```diff
@@ -3,17 +3,20 @@ import PropTypes from 'prop-types';
 import { View, TextInput, Text } from 'react-native';
 import styles from './styles';
 
-const Input = ({ input: { onChange, ...restInput }, password = false, label, meta: { touched, error } }) => (
+const Input = ({ input: { onChange, ...restInput }, placeholder = '', autoCapitalize = 'none', password = false, keyboardType = 'default', label, meta: { touched, error } }) => (
   <View>
-    {label && <Text>{label}</Text>}
+    {label && <Text style={styles.label}>{label.toUpperCase()}</Text>}
     <View>
       <TextInput
-        style={styles.input}
+        style={(touched && error) ? styles.inputError : styles.input}
         onChangeText={onChange}
         secureTextEntry={password}
+        placeholder={placeholder}
+        keyboardType={keyboardType}
+        autoCapitalize={autoCapitalize}
         {...restInput}
       />
-      {touched && error && <Text>{error}</Text>}
+      <Text style={styles.errorLabel}>{(touched && error) ? error : ''}</Text>
```

Comment:
Here you moved the condition to the text part, so the text tag is always rendered. I'm not sure if you wanted this, but normally if you don't need to display its content, you won't display the tag either. I would move the condition to the outside:

`{touched && error && <Text style={styles.errorLabel}>{error}</Text>}`

Reply:
I did this because I wanted the error tag to occupy the space whether it is showing an error or not. Is there any way to accomplish that in a better way?

Reply:
That's fine then. What you wanted is odd 😂

Reply:
The designs are like that 🤷‍♂️